src/preferences/dialog/dlgprefinterfacedlg.ui

```diff
@@ -247,6 +224,46 @@
      </layout>
     </widget>
    </item>
+   <item row="1" column="0">
+    <widget class="QGroupBox" name="groupBoxEffectOption">
+     <property name="title">
+      <string>Effect options</string>
+     </property>
+     <layout class="QGridLayout" name="gridLayout">
+      <item row="0" column="0">
+       <widget class="QLabel" name="labelLoadBehavior">
+        <property name="text">
+         <string>Load behavior</string>
+        </property>
+       </widget>
+      </item>
+      <item row="0" column="1" colspan="2">
+       <layout class="QHBoxLayout" name="horizontalLayoutEffects">
+        <item>
+         <widget class="QRadioButton" name="radioButtonKeepMetaknobPosition">
+          <property name="text">
+           <string>Keep metaknob position</string>
```

Comment:
Keeping the position is not the most important thing.
It is that the effect parameter are forced to the metaknob value instead of the default.
That's why I prefer "Adopt metaknob value" or any other short phrase that expresses this.

Reply:
I do not think people who have not been following this discussion will understand those implementation details. What they will see is whether the metaknob moves or not when they switch effects.

Reply:
Ok

Reply:
A more detailed description might help:
[keep/preserve/respect] metaknob position and [adopt/adjust/apply to] effect parameters
Please choose whatever wording fits best here, since I'm not a native speaker.

Technically it is "loading", but the actual use case is "switching" from one effect to another. "Loading and switching effects" instead of "Load behaviour" might be easier for identifying the use case.

Reply:
Those are too many words for the preferences UI:
![image](https://user-images.githubusercontent.com/9455094/38246391-420142ac-3707-11e8-93a1-26e9cf060862.png)